### ISSUE TITLE
Invisible text after opening Windows Terminal by double-clicking .bat file

### ISSUE THREAD

**Issue by n0samu** [microsoft/terminal#18259]
### Windows Terminal version

1.21.3231.0

### Windows build number

10.0.19045.5131

### Steps to reproduce

1. Make sure the Launch Size setting is set to 120 columns and Windows Terminal is set as the default terminal application. Make sure that Windows Terminal opens when you double-click a .bat file.
2. Create a .bat file with the following code:
```bat
@ECHO OFF
CLS & ECHO.
ECHO ------------------------------------------------------------------------------------------------------------------------ 
ECHO.
SET /P INPUT="Enter a filename or URL: "
```
3. Close any open terminal windows, then double-click the .bat file to run it.

### Expected Behavior

Here's what I see in my terminal window if I run the .bat file from an existing terminal session and type "my input" at the prompt:
![Image](https://github.com/user-attachments/assets/d4cdccdb-2858-485e-a97b-53b9a7b60460)

### Actual Behavior

When I double-click the .bat file to run it in a new terminal window, the prompt text is invisible and the input text appears slightly above where it should:
![Image](https://github.com/user-attachments/assets/478d56b3-2c03-4468-b8cc-670b9e431370)

I have also observed the input text being shifted upward while the prompt text is visible:
![Image](https://github.com/user-attachments/assets/dc50a953-dd7b-4487-a9d0-f1c9652976b5)

**Comment by ztcarm** [comment]
What os are you using?

**Comment by j4james** [comment]
I'm not sure what the problem is, but I think this may be fixed already. I can reproduce it in an old release version of Windows Terminal that I still have installed (v1.20.11781), but not in the preview build (v1.22.2702).

**Comment by n0samu** [comment]
Thanks, I think you're right. The problem is not reproducible for me in Terminal Preview 1.22.3232.0.

**Comment by carlos-zamora** [comment]
Great! Since it's fixed in v1.22, I'm going to close this. Thanks!
